Libraries/Text/TextInput/RCTBaseTextInputShadowView.m

```diff
@@ -218,7 +218,16 @@ - (NSAttributedString *)measurableAttributedText
 
 - (CGSize)sizeThatFitsMinimumSize:(CGSize)minimumSize maximumSize:(CGSize)maximumSize
 {
-  NSAttributedString *attributedText = [self measurableAttributedText];
+  NSMutableAttributedString *attributedText = [[self measurableAttributedText] mutableCopy];
```

Comment:
I think it might be worth adding a comment that explains that ios doesnt support setting fixed number of lines and what we are doing.

Reply:
Same for the fabric part

Reply:
Added! Let me know if the comments make sense.

Reply:
Is there a reason for adding the new lines before the text instead of after? Is there any cases where it would affect the width of the input? Also the paper implementation doesn’t need the extra K char to work? I also wonder if there would be a better character to use which has 0 width

Reply:
I think adding after should work well too.